### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by phil-flip** [rr-/szurubooru#513] [General]
I kind of feel like there is nothing much going on in this project.
So unless anyone has an active fork to link or the project breaths back to life, I am going to work it myself and try to maintain PRs and issues on a future Discord server.
Because right now, issues are piling up and there is no real sign from the original developers for some time now.

**Comment by neobooru** [comment]
In my opinion it is kinda hard to maintain this project and/or add new features to it. Both the frontend and the backend use a custom made framework, and not something like django + some frontend JS framework. This makes it difficult for me (and anyone else) to add new functionality too it.

Furthermore, I don't feel comfortable adding hacky code in a project that is not my own. [Take this PR as example.](https://github.com/rr-/szurubooru/pull/445/files) The code works fine, I've been using it in my fork for a while now. The code quality however is not something to write home about, which is why I'm reluctant to merge it until I've fixed it. Unfortunately I've never found the time to actually improve it, hence it just lying there.

Because of the above two points I myself very rarely make something which I feel is merge-worthy. The core functionality is 'good enough' so I never really get the urge that I _need_ to develop a new feature. And when I feel like it is missing a feature then it is usually easier to create a second application to fill that gap, e.g. my own [SzuruChrome](https://github.com/neobooru/SzuruChrome) and [reluce's szurubooru-toolkit](https://github.com/reluce/szurubooru-toolkit). Both of those implement "missing" features from szurubooru, and it is usually easier to add them in a standalone app. I am not saying that it is better that way, just easier for the developers.

There are some forks with more activity: for example [Hunternif's fork](https://github.com/Hunternif/szurubooru) or [skybldev's fork](https://github.com/skybldev/szurubooru_photos). Unfortunately I just don't have the time to review all their changes and merge them back in this project.

Lastly, [rr-](https://github.com/rr-) pretty much stepped down from the project, so you probably won't seem them working on anything relating this project.

**Comment by phil-flip** [comment]
seems like some additional Fork has been popping up with a Rust backend: https://github.com/liamw1/oxibooru
